### ISSUE TITLE
TypeError when creating the first Orderable object

### ISSUE THREAD

**Issue by Sam-Costigan** [elton2048/wagtail-orderable#21]
Great package, thanks a lot for this.

I have just discovered that when saving the first object to an otherwise empty table of Orderable objects, the following `TypeError` will be thrown:
```
TypeError at /admin/app/accessibility/create/
unsupported operand type(s) for +: 'NoneType' and 'int'
```
This is because the following queryset in the `Orderable().save()` method will return `None` when it is working with an empty queryset:
```python
self.__class__.objects.aggregate(Max('sort_order'))['sort_order__max']
```
All that is needed to fix this is a fallback to 0 when the queryset is empty. This might be as simple as the following:
```python
# models.py/Orderable
def save(self, *args, **kwargs):
    if self.pk is None:
        sort_order_max = self.__class__.objects.aggregate(Max('sort_order'))['sort_order__max'] or 0
        self.sort_order = sort_order_max + 1
    super().save(*args, **kwargs)
```
I'd be happy to put together a PR for this change.

**Comment by theRustyKnife** [comment]
As this doesn't seem like it's going to be fixed anytime soon, I figured I'd share my workaround:

Make an app in your project called `wagtailorderable_patch` and paste this into it's `models.py`:
```python
from django.db.models import Max

from wagtailorderable.models import Orderable


def patch_orderable():
    def save(self, *args, **kwargs):
        if self.pk is None:
            self.sort_order = (self.__class__.objects.aggregate(Max('sort_order'))['sort_order__max'] or 0) + 1
        super(Orderable, self).save(*args, **kwargs)

    Orderable.save = save


patch_orderable()
```

Then make sure to add `wagtailorderable_patch` to your `INSTALLED_APPS` **after** `wagtailorderable` and that's it.

**Comment by victoriachan** [comment]
Thanks, @theRustyKnife for your patch.

**Comment by elton2048** [comment]
Fixed by #23 in master branch. Will rollout new version for that

**Comment by elton2048** [comment]
Release 1.0.4 is there!
